libsolidity/interface/StorageLayout.cpp

```diff
@@ -46,6 +46,7 @@ Json StorageLayout::generate(ContractDefinition const& _contractDef)
 
 Json StorageLayout::generate(VariableDeclaration const& _var, u256 const& _slot, unsigned _offset)
 {
+	solUnimplementedAssert(_var.referenceLocation() != VariableDeclaration::Location::Transient, "");
```

Comment:
Just a reminder that these messages have been optional for quite a while now:

```suggestion
	solUnimplementedAssert(_var.referenceLocation() != VariableDeclaration::Location::Transient);
```

No need to remove them from existing code, but when writing new code it's cleaner not to add the empty message.